.env.example

```
@@ -0,0 +1,212 @@
+# Log levels: error, warn, info, debug
+LOG_LEVEL=info
+LOG_FILE=./logs/app.log
+LOG_MAX_SIZE=10m
+LOG_MAX_FILES=5
+
+# =============================================================================
+# BACKUP CONFIGURATION
+# =============================================================================
+# Database backup
+BACKUP_ENABLED=true
+BACKUP_SCHEDULE=0 2 * * *
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)